### PR TITLE
Disabled rotation and small fixes

### PR DESCRIPTION
**Pull request by frnzee**
- Units don't rotate at Y angle now to make it look better
- Units don't shoot when the enemy is dead or their health is below zero
- Small semantic fixes

### CODE REVIEW

File: Assets/Scripts/HealthBar.cs [hunk 1]
```diff
@@ -24,14 +27,30 @@ private void Update()
...
+            _healthText.text = "DEAD";
+        }
+        else
+        {
+            _healthText.text = Mathf.RoundToInt(_unitStats.Health).ToString();
+        }
+
+        if (_unitStats.Health >= _unitStats.MaxHealth)
+        {
+            _healthBar.SetActive(false);
```
Comment (ega-ega) [inline Assets/Scripts/HealthBar.cs:41]: You can simplify these lines: _healthBar.SetActive(_unitStats.Health < _unitStats.MaxHealth);

File: Assets/Scripts/Warrior.cs [hunk 2]
```diff
@@ -131,12 +134,15 @@ private void Shooting()
     {
         _shootingTimer -= Time.deltaTime;
 
-        if (!IsDead && _isShooting && _shootingTimer <= 0)
+        if (_currentEnemy.UnitStats.Health >= 0)
```
Comment (ega-ega) [inline Assets/Scripts/Warrior.cs:137]: Create a property IsDead(bool) in UnitStats.
And return true if lives are zero or below
And check it when you check if the enemy is alive.

This approach hides the implementation of death in the UnitStats class. and thanks to this, the principle of encapsulation is realized. https://en.wikipedia.org/wiki/Encapsulation_(computer_programming)

**Review by ega-ega — CHANGES_REQUESTED** [review]

File: Assets/Scripts/HealthBar.cs (hunk 1 above)
Comment (frnzee) [reply Assets/Scripts/HealthBar.cs]: Done

File: Assets/Scripts/Warrior.cs (hunk 2 above)
Comment (frnzee) [reply Assets/Scripts/Warrior.cs]: Done

**Review by ega-ega — APPROVED** [review]

**Pull request merged**